bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/server-channel-groups.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="freeboxos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="sensors">
+		<label>System Sensors</label>
+	</channel-group-type>
+
+	<channel-group-type id="sysinfo">
+		<label>System Informations</label>
+		<channels>
+			<channel id="uptime" typeId="uptime"/>
+			<channel id="box-event" typeId="box-event"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="actions">
+		<label>Server Settings</label>
+		<channels>
+			<channel id="wifi-status" typeId="wifi-status"/>
+			<channel id="ftp-status" typeId="ftp-status"/>
+			<channel id="upnpav-status" typeId="upnpav-status"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="samba">
+		<label>Windows File Sharing</label>
+		<channels>
+			<channel id="fileshare-status" typeId="fileshare-status"/>
+			<channel id="printershare-status" typeId="printershare-status"/>
+		</channels>
+	</channel-group-type>
```

Comment:
I would suggest to do like in the Freebox WEB app, that is having a group named "File Sharing" including the channel ftp-status.

Reply:
Very valid point.